desktop/core/src/desktop/lib/raz/raz_client.py

```diff
@@ -144,17 +142,47 @@ def check_access(self, method, url, params=None, headers=None, data=None):
   def _handle_raz_req(self, raz_url, request_headers, request_data):
     if self.auth_type == 'kerberos':
       auth_handler = requests_kerberos.HTTPKerberosAuth(mutual_authentication=requests_kerberos.OPTIONAL)
-      raz_req = requests.post(raz_url, headers=request_headers, json=request_data, auth=auth_handler, verify=False)
+      raz_response = self._handle_raz_ha(raz_url, headers=request_headers, data=request_data, auth_handler=auth_handler)
```

Comment:
Could we use `raz_urls` instead of `raz_url` for improved readability, especially within the context of the `handle_raz_ha` function?

Reply:
Changed it to `raz_url` in the ha method also.